src/main/scala/com/softwaremill/react/kafka/ConsumerProps.scala

```diff
@@ -1,7 +1,9 @@
-package kafka.consumer
+package com.softwaremill.react.kafka
 
 import java.util.Properties
-import java.util.UUID
+
+import kafka.consumer.ConsumerConfig
+import kafka.serializer.Decoder
 
 object ConsumerProps {
```

Comment:
using `Props` may be a bit confusing here (because of akka `Props`). Maybe rename to `ConsumerProperties`? (similar for `ProducerProps`)